src/CattyTests/DataModel/VariablesContainerTest.swift

```diff
@@ -459,6 +459,32 @@ final class VariablesContainerTest: XCTestCase {
         XCTAssertFalse(container.isProjectVariableOrList(list2))
     }
 
+    func testIsProjectVariable() {
+        let userVariable1 = UserVariable(name: "testVar1", isList: false)
+        let userVariable2 = UserVariable(name: "testVar2", isList: false)
+
+        let container = VariablesContainer()
+
+        container.programVariableList.add(userVariable1)
+        container.objectVariableList.setObject(userVariable1, forKey: "VariableKey")
+
+        XCTAssertTrue(container.isProjectVariable(userVariable1))
+        XCTAssertFalse(container.isProjectVariable(userVariable2))
+    }
+
+    func testIsProjectList() {
+        let list1 = UserList(name: "testList1")
+        let list2 = UserList(name: "testList2")
+
+        let container = VariablesContainer()
+
+        container.programListOfLists.add(list1)
+        container.objectListOfLists.setObject(list2, forKey: "listKey")
```

Comment:
Please make use of the function `addObjectVariable` instead of adding them to the data structure directly.